docs/sections/pipeline_samples/tutorials/GenerateSentencePair.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<p>Even if in markdown, better show it using <code></code>`python</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/893/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>